scripts/bootstrap.sh.in

```diff
@@ -53,11 +65,14 @@ trap cleanup EXIT
 BASE_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
 
 RPM=${RPM:-$(command -v rpm)}
```

Comment:
Need to be replaced by:
```RPM=${RPM:-$(command -v rpm || true)}```
Otherwise it'll fail.

Reply:
Indeed, to avoid the fails for my tests, I comment the `set -e` thank you for the reminder !